docs/user-guide/plugins.md

```diff
@@ -104,7 +104,11 @@ plugin "foo" {
 }
 ```
 
-When the plugin is enabled, TFLint invokes the `tflint-ruleset-[name]` (`tflint-ruleset-[name].exe` on Windows) binary in the plugin directory (For instance, `~/.tflint.d/plugins/tflint-ruleset-[name]`). So you should move the binary into the directory in advance.
+When enabled, TFLint looks for the plugin binary following the pattern tflint-ruleset-\[name] (or tflint-ruleset-\[name].exe on Windows) directly in your plugins folder (e.g., ~/.tflint.d/plugins/ on Unix-like systems or C:\Users\\[username]\\.tflint.d\plugins\ on Windows).
+
+Thus, with the configuration above where the plugin name is "foo", the executable must be named tflint-ruleset-foo (or tflint-ruleset-foo.exe on Windows). So you should move the binary into the plugin directory in advance.
+
+**Important for Windows Users:** On Windows, make sure to place the executable directly into the plugins folder (e.g. C:\Users\\[username]\\.tflint.d\plugins\) without using any subfolders. This direct placement is required for TFLint to detect and load the plugin correctly.
```

Comment:
Is this sentence necessary? I don't think there is any difference between Windows and Unix, except for the `.exe` extension in the file name.

Reply:
Personally, I think this sentence should be removed altogether.

I understand that your misunderstanding arose because it was unclear in the original sentence whether "(For instance, `~/.tflint.d/plugins/tflint-ruleset-[name]`)" referred to "the plugin directory" or "binary in the plugin directory".

Since the revised sentence does not contain this ambiguity, I do not think it is necessary to emphasize this issue so much.